node/name-service/main.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+const beeline = require('honeycomb-beeline')({
```

Comment:
https://docs.honeycomb.io/getting-data-in/javascript/beeline-nodejs/#interoperability-with-opentelemetry

Reply:
Need to add w3c interop, so the beeline plays nice with OTEL (trace interop)